platform/os_windows.go

```diff
@@ -391,8 +397,8 @@ func tryStopServiceFn(ctx context.Context, service managedService) func() error
 				break
 			}
 			select {
-			case <-ctx.Done():
-				return errors.Wrap(ctx.Err(), "context cancelled")
+			case <-deadline.Done():
+				return errors.Wrap(deadline.Err(), "context cancelled")
```

Comment:
nit: deadline exceeded

Reply:
unwapped, deadline error has suffifient context in the message already